### ISSUE TITLE
rawgit shutting down; update links

### ISSUE THREAD

**Issue by abhishalya** [coala/projects#749]
rawgit is shutting down and bears are throwing errors like [these](https://travis-ci.org/coala/projects/jobs/576470295#L700).

**Comment by jayvdb** [comment]
If the links can be easily replaced with an alternative, that would be great.

Otherwise an ignore entry in .coafile is needed now, and https://github.com/coala/coala-bears/issues/2952 is an idea for how we could provide a way to allow broken links to be shown as broken links.

**Comment by abhishalya** [comment]
Can't we directly use links from github like https://rawgit.com/saksham189/GSoC-Report/master/work.tar.gz?

**Comment by abhishalya** [comment]
>Can't we directly use links from github like https://github.com/saksham189/GSoC-Report/raw/master/work.tar.gz or maybe https://raw.githubusercontent.com/coala/coalaCSS/master/coala.css?

Okay, well [this](https://github.com/rgrove/rawgit/blob/master/FAQ.md#why-is-this-necessary-cant-i-just-load-files-from-github-directly) explains that. But I don't think it affects us too much. But, still if we need something like rawgit, we could use https://raw.githack.com/ which seems to be the exact same thing.

**Comment by KVGarg** [comment]
> we could use https://raw.githack.com/ which seems to be the exact same thing.

Yeah! We can use that too. Also there were some following free services suggested by rawgit on there website that offer fantastic alternatives to some or all of RawGit's functionality. You may like them even more than RawGit.

- jsDelivr
- GitHub Pages
- CodeSandbox
- unpkg

**Comment by moazfarrukh** [comment]
has this issue been resolved?
